<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralDataFittingViewControls</class>
 <widget class="QWidget" name="GeneralDataFittingViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>1124</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Selected Time Series:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QmitkSingleNodeSelectionWidget" name="timeSeriesNodeSelector" native="true"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Selected Mask:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QmitkSingleNodeSelectionWidget" name="maskNodeSelector" native="true"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkMaskInfo">
       <property name="text">
        <string>Show mask info</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMaskInfo">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please only use static segmentation masks as input in order to get sensible parametric map results.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Fitting strategy</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioPixelBased">
          <property name="text">
           <string>Pixel based</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioROIbased">
          <property name="text">
           <string>ROI based</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="comboModel">
     <item>
      <property name="text">
       <string>Select fitting model...</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupGeneric">
     <property name="title">
      <string>Generic model settings:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lableFormula">
          <property name="text">
           <string>Formula:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="editFormula"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lableNrOfParams">
          <property name="text">
           <string>Number of parameters:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="nrOfParams">
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkFormulaInfo">
        <property name="text">
         <string>Show formula info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFormulaInfo">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You may evaluate/fit simple mathematical formulas (e.g. &amp;quot;&lt;span style=&quot; font-style:italic;&quot;&gt;3.5 + a * x * sin(x) - 1 / 2&lt;/span&gt;&amp;quot;).&lt;/p&gt;&lt;p&gt;The parser is able to recognize:&lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;sums, differences, products and divisions (a + b, 4 - 3, 2 * x, 9 / 3)&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;algebraic signs ( +5, -5)&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;exponentiation ( 2 ^ 4 )&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;parentheses (3 * (4 + 2))&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;following unary functions: &lt;span style=&quot; font-style:italic;&quot;&gt;abs&lt;/span&gt;, &lt;span style=&quot; font-style:italic;&quot;&gt;exp&lt;/span&gt;, &lt;span style=&quot; font-style:italic;&quot;&gt;sin&lt;/span&gt;, &lt;span style=&quot; font-style:italic;&quot;&gt;cos&lt;/span&gt;, &lt;span style=&quot; font-style:italic;&quot;&gt;tan&lt;/span&gt;, &lt;span style=&quot; font-style:italic;&quot;&gt;sind&lt;/span&gt; (sine in degrees), &lt;span style=&quot; font-style:italic;&quot;&gt;cosd&lt;/span&gt; (cosine in degrees), &lt;span style=&quot; font-style:italic;&quot;&gt;tand&lt;/span&gt; (tangent in degrees)&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;variables (x, a, b, c, ... , j)&lt;/li&gt;&lt;/ul&gt;&lt;p&gt;Remark: The variable &amp;quot;x&amp;quot; is reserved. It is the signal position / timepoint. Don't use it for a model parameter that should be deduced by fitting (these are represented by a..j).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_FitConfiguration">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Model Fit Configuration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QToolBox" name="toolboxConfiguration">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QWidget" name="groupStartParameter">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>528</width>
           <height>155</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Start parameter</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QRadioButton" name="radioButton_StartParameters">
            <property name="text">
             <string>Enter Fit Starting Parameters manually</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QmitkInitialValuesManagerWidget" name="initialValuesManager" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="groupConstraints">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>528</width>
           <height>232</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Constraints</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QCheckBox" name="checkBox_Constraints">
            <property name="text">
             <string>Enter Constraints for Fit Parameters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QmitkSimpleBarrierManagerWidget" name="constraintManager" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="topMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Fitting name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineFitName">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Name/prefix that should be used for the fitting results.&lt;/p&gt;&lt;p&gt;May be explicitly defined by the user.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="placeholderText">
        <string>default fit name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnModelling">
     <property name="text">
      <string>Start Modelling</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkDebug">
     <property name="text">
      <string>Generate debug parameter images</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="infoBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkSimpleBarrierManagerWidget</class>
   <extends>QWidget</extends>
   <header>QmitkSimpleBarrierManagerWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkInitialValuesManagerWidget</class>
   <extends>QWidget</extends>
   <header>QmitkInitialValuesManagerWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkSingleNodeSelectionWidget</class>
   <extends>QWidget</extends>
   <header>QmitkSingleNodeSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
